test/rules.js

```diff
@@ -12,3 +12,50 @@ test('badge - incorrect source', async t => {
 	t.is(result.ruleId, 'awesome-badge');
 	t.is(result.message, 'Incorrect badge source');
 });
+
+test('list-item - incorrect item prefix', async t => {
+	t.plan(2);
+	let messages = (await m({filename: 'fixtures/list-item/1.md'})).messages;
+	messages = messages.filter(message => message.ruleId === 'awesome-list-item');
+	messages.forEach(message => t.is(message.message, 'List items must start with `- [name](link)`'));
+});
+
+test('list-item - missing dash between link and description', async t => {
+	t.plan(1);
```

Comment:
I was supposed to ask you this but I forgot: so instead of `t.plan(2)` is better to check the length of the `messages`?